app/assets/stylesheets/application.scss

```diff
@@ -215,14 +245,18 @@
 //   height: 25%;
 // }
 
+h2.left-pad-me{
```

Comment:
Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space